src/dsp_tools/utils/xmlupload/upload_stashed_resptr_props.py

```diff
@@ -97,6 +74,125 @@ def upload_stashed_resptr_props(
     return nonapplied_resptr_props
 
 
+def _upload_all_resptr_props_of_single_resource(
+    resource_in_triplestore: dict[str, Any],
+    link_prop: XMLProperty,
+    res_iri: str,
+    stashed_resource: XMLResource,
+    resptrs: list[str],
+    id2iri_mapping: dict[str, str],
+    con: Connection,
+    nonapplied_resptr_props: dict[XMLResource, dict[XMLProperty, list[str]]],
+    verbose: bool,
+) -> dict[XMLResource, dict[XMLProperty, list[str]]]:
+    """
+    This function takes one resource stashed resource and resptr-props that are specific to a property.
+    It uploads them to the triplestore and removes them from the nonapplied_resptr_props dictionary.
```

Comment:
We don't have access to the database (=triplestore), we always send a request to DSP-API, which in turn writes into the database.

```suggestion
    This function takes one stashed resource and the resptr-props that are specific to a property.
    It uploads them to DSP-API and removes them from the nonapplied_resptr_props dictionary.
```

Reply:
Done